### ISSUE TITLE
Misplaced wing textures

### ISSUE THREAD

**Issue by philfsx21** [flybywiresim/a32nx#8071]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2023-06-09T12:58:01+00:00",
    "ref": "refs/heads/master",
    "sha": "e7c4fa0ff6e22cc1268abb30f82f28372d301267",
    "actor": "tracernz",
    "event_name": "manual",
    "pretty_release_name": "master:e7c4fa0f",
    "version": "v0.10.0-dev.e7c4fa0"
}
```

### Describe the bug

I noticed that the emergency exits (overwing) and the textures on the wings (red lines) are not aligned. It seems like the red lines are simply misplaced. I encountered this issue with all downloaded liveries.

![window_exits_fbw](https://github.com/flybywiresim/a32nx/assets/136201770/a7626c8a-8a08-49d5-bf48-4597a0728ada)

### Expected behavior

Please consider editing the basic textures of the plane (-> moving the red lines on the wing forward)  so that the plane is even more realistic in the future. :)

### Steps to reproduce

1. Start MSFS and start a flight with any FBW A320neo livery.
2. Go into external view and look at the relation between the overwing exits and the red lines on the wing.
3. Compare with real pictures of the aircraft/wings. 

### References (optional)

![window_exits_fbw](https://github.com/flybywiresim/a32nx/assets/136201770/0e1866c6-dc88-4c66-9b3a-1112757bb4df)

-> You can clearly see that the red emergency pathway should be moved forward.

**Comment by tracernz** [comment]
I don't think this is really fixable (without large effort) with the Asobo model. It's a combination of the black lines on the wing, the exit decals, and the windows all being a little off. Will have to wait for the FBW model.
